packages/stark-ui/tsconfig.lib.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.json",
+  "angularCompilerOptions": {
+    "skipTemplateCodegen": true,
+    "strictMetadataEmit": true,
+    "fullTemplateTypeCheck": true,
+    "enableResourceInlining": true
+  },
+  "outDir": "dist",
+  "buildOnSave": false,
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "AUTOGENERATED",
+    "declarationDir": "AUTOGENERATED",
+    "inlineSourceMap": true
+  },
+  "exclude": ["node_modules", "dist", "**/*.ngfactory.ts", "**/*.shim.ts", "**/*.spec.ts"],
```

Comment:
Why are `**/*.ngfactory.ts` and `**/*.shim.ts` included here? We have no such files